newscoop/install/Resources/sql/upgrade/4.4.x/2016.05.04/tables.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `Cache` ENGINE = InnoDB;
```

Comment:
yeah, I will add that, in case there is wrong (truncated) content already cached